R/utils.R

```diff
@@ -173,3 +173,7 @@ as_map <- function(x) {
 
   x
 }
+
+scrub_environment <- function(x) {
```

Comment:
IIUC it is only used when comparing a named to an unnamed environment right?

I wonder if there's a better way to convey this than ???? (which to me implies "unknown" rather than "hidden")

what about "<unnamed environment $ADDR>"

Reply:
I made the most minimal change I could think of: replacing `?` with `*` to implied "starred out".